tools/cdk-build-tools/config/jest.config.json

```diff
@@ -1,7 +1,12 @@
 {
   "moduleFileExtensions": [
+    "ts",
     "js"
```

Comment:
Let's get rid of `js` if we're testing `ts`. Doesn't this lead to testing every file twice?

Reply:
Surprisingly, that doesn't work:

```
@aws-cdk/cx-api: ● Validation Error:
@aws-cdk/cx-api:   moduleFileExtensions must include 'js':
@aws-cdk/cx-api:   but instead received:
@aws-cdk/cx-api:     ["ts"]
@aws-cdk/cx-api:   Please change your configuration to include 'js'.
@aws-cdk/cx-api:   Configuration Documentation:
@aws-cdk/cx-api:   https://jestjs.io/docs/configuration.html

```